packages/default/scss/orgchart/_layout.scss

```diff
@@ -0,0 +1,92 @@
+@include exports("orgchart/layout") {
+
+    // OrgChart
+    .k-orgchart {
+        width: 100%;
+        box-sizing: border-box;
+        font-size: $orgchart-font-size;
+        font-family: $orgchart-font-family;
+        line-height: $orgchart-line-height;
+        display: flex;
+        justify-content: center;
+        overflow: auto;
+    }
+
+    .k-orgchart-container {
+        position: relative;
+    }
+
+    // Nodes
+    .k-orgchart-group {
+        gap: $orgchart-group-gap;
+    }
+
+    .k-orgchart-group.k-vstack > .k-orgchart-node-container {
+        gap: 0;
```

Comment:
why does the vertical group differ? what's so special about it?

Reply:
The nodes in the vertical groups are connected with lines, thus having gap between them is not applicable.